sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -411,9 +424,14 @@ public PDone apply(PCollection<T> input) {
 
     @Override
     public void validate(PCollection<T> input) {
-      checkNotNull(getDataSourceConfiguration(), "dataSourceConfiguration");
-      checkNotNull(getStatement(), "statement");
-      checkNotNull(getPreparedStatementSetter(), "preparedStatementSetter");
+      checkArgument(getDataSourceConfiguration() != null,
+          "JdbcIO.write().withDataSourceConfiguration(dataSourceConfiguration) called with null "
```

Comment:
Yes, indeed. I will "extend" the message with `JdbcIO.write().withDataSourceConfiguration(configuration) not called or called with null`.

Thanks.